CONFIG_REFERENCE.md

```diff
@@ -194,25 +197,56 @@ specify them explicitly!)
   - `src_hostname`
     Hostname reported by the client in the EHLO/HELO command.
 
-  See below for example.
+    Example:
+    ```
+    match rcpt_domain emersion.fr {
+        filter dkim verify
+        deliver local
+    }
+    ```
 
+* `destination <recipient...> { ... }`
```

Comment:
Thinking about it a little bit more, maybe we should name it `fork` or `split`? Not sure

Reply:
Does `destination` documentation sound like it splits something? I'm trying to hide the internal complexity of implementation, so the user will need to be aware of it only in cases where tight control of message dispatching is required.

Reply:
Okay, `destination` is probably better. However I don't think the documentation is clear enough. IMHO it should say something among the lines of: for all matching recipients execute the block and stop processing, for all others continue processing.

We may also want to make it mandatory to have `deliver` as the last line of a `destination` block.

Reply:
Yeah, that is much more explicit. 
Now it says:
```
  For all recipients that match at least one rule - execute subblock and stop
  processing, for all others - skip block and continue.
```

Reply:
LGTM!